services/worker-manager/src/api.js

```diff
@@ -195,3 +227,38 @@ builder.declare({
   }
   return res.reply(data);
 });
+
+/*
+ * ************** BELOW HERE LIVE PROVIDER ENDPOINTS **************
+ */
+
+builder.declare({
```

Comment:
so, each provider will have its own API? What sort of endpoints will there be?

Reply:
I think we'll want to include the providerId in here, so if for example we have two gcp providers configured they're not fighting over `/credentials/google/:name`

Reply:
what could we use two gcp providers for?

Reply:
Ah, making the provider an arg makes sense too I think.

We would have multiple gcp providers for a couple potential reasons:

1. If we end up having a second version of a google provider once we learn more about how this one fits our needs. We would run both for a bit while we transition workers over.
2. Each provider only handles one project. In production we'll actually have multiple projects so that level-3 is isolated from level-1 or servo people are isolated from gecko people, etc.